src/map/storage.c

```diff
@@ -356,12 +391,26 @@ int storage_storagegettocart(struct map_session_data* sd, int index, int amount)
  *------------------------------------------*/
 void storage_storageclose(struct map_session_data* sd)
 {
+	int i = 0;
+
 	nullpo_retv(sd);
 
+	Assert_retv(sd->state.storage_flag == STORAGE_FLAG_NORMAL);
```

Comment:
There are some cases that call this function just to ensure that the storage is saved, even if it's not open in the client (item check, `@clearstorage`). This assertion would fail